linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/tests/user_password_change_time_in_future.fail.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# remediation = none
+# packages = passwd
```

Comment:
```suggestion
```
Now we can remove this header since the `passwd` command is no longer used.

Reply:
👍